<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KSerialDeviceEmulatorWidgetBase</class>
  <widget class="QWidget" name="kserialdeviceemulatorwidgetbase" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>677</width>
        <height>790</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>kserialdeviceemulatorwidgetbase</string>
    </property>
    <layout class="QGridLayout" >
      <item row="0" column="0" >
        <layout class="QHBoxLayout" >
          <item>
            <widget class="QLabel" name="textLabel1" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>1</hsizetype>
                  <vsizetype>1</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text" >
                <string>Log File</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="KUrlRequester" name="u_logfile" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>1</hsizetype>
                  <vsizetype>1</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
            </widget>
          </item>
          <item>
            <widget class="KPushButton" name="b_loadlog" >
              <property name="text" >
                <string>Lo&amp;ad</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item row="1" column="0" >
        <widget class="QSplitter" name="splitter5" >
          <property name="orientation" >
            <enum>Qt::Vertical</enum>
          </property>
          <widget class="QSplitter" name="splitter3" >
            <property name="orientation" >
              <enum>Qt::Horizontal</enum>
            </property>
            <widget class="K3ListView" name="cmdListView" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>7</hsizetype>
                  <vsizetype>7</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="allColumnsShowFocus" >
                <bool>true</bool>
              </property>
              <property name="fullWidth" >
                <bool>true</bool>
              </property>
              <column>
                <property name="text" >
                  <string>Index</string>
                </property>
                <property name="clickable" >
                  <bool>true</bool>
                </property>
                <property name="resizable" >
                  <bool>true</bool>
                </property>
              </column>
              <column>
                <property name="text" >
                  <string>Command</string>
                </property>
                <property name="clickable" >
                  <bool>true</bool>
                </property>
                <property name="resizable" >
                  <bool>true</bool>
                </property>
              </column>
              <column>
                <property name="text" >
                  <string>Answer</string>
                </property>
                <property name="clickable" >
                  <bool>true</bool>
                </property>
                <property name="resizable" >
                  <bool>true</bool>
                </property>
              </column>
            </widget>
            <widget class="QWidget" name="layout3" >
              <layout class="QGridLayout" >
                <item row="1" column="0" >
                  <widget class="KPushButton" name="b_remove" >
                    <property name="sizePolicy" >
                      <sizepolicy>
                        <hsizetype>1</hsizetype>
                        <vsizetype>1</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text" >
                      <string>R&amp;emove</string>
                    </property>
                  </widget>
                </item>
                <item rowspan="1" row="0" column="0" colspan="2" >
                  <widget class="QLabel" name="cmdDesc" >
                    <property name="sizePolicy" >
                      <sizepolicy>
                        <hsizetype>5</hsizetype>
                        <vsizetype>3</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text" >
                      <string>Raw command</string>
                    </property>
                    <property name="alignment" >
                      <set>Qt::AlignTop</set>
                    </property>
                    <property name="wordWrap" >
                      <bool>true</bool>
                    </property>
                  </widget>
                </item>
                <item row="1" column="1" >
                  <widget class="KPushButton" name="b_removeall" >
                    <property name="sizePolicy" >
                      <sizepolicy>
                        <hsizetype>1</hsizetype>
                        <vsizetype>1</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text" >
                      <string>Remo&amp;ve All</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </widget>
          </widget>
          <widget class="QWidget" name="layout5" >
            <layout class="QVBoxLayout" >
              <item>
                <layout class="QHBoxLayout" >
                  <item>
                    <widget class="KComboBox" name="eventBox" >
                      <property name="sizePolicy" >
                        <sizepolicy>
                          <hsizetype>3</hsizetype>
                          <vsizetype>0</vsizetype>
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                        </sizepolicy>
                      </property>
                      <property name="editable" >
                        <bool>true</bool>
                      </property>
                      <property name="currentItem" >
                        <number>0</number>
                      </property>
                      <property name="insertionPolicy" >
                        <enum>NoInsertion</enum>
                      </property>
                      <property name="duplicatesEnabled" >
                        <bool>false</bool>
                      </property>
                      <item>
                        <property name="text" >
                          <string/>
                        </property>
                      </item>
                      <item>
                        <property name="text" >
                          <string>Incoming Call</string>
                        </property>
                      </item>
                    </widget>
                  </item>
                  <item>
                    <widget class="KPushButton" name="b_event" >
                      <property name="text" >
                        <string>&amp;Send Event</string>
                      </property>
                    </widget>
                  </item>
                </layout>
              </item>
              <item>
                <widget class="Q3TextBrowser" name="tb_rawlog" />
              </item>
            </layout>
          </widget>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <customwidgets>
    <customwidget>
      <class>KComboBox</class>
      <extends>QComboBox</extends>
    </customwidget>
    <customwidget>
      <class>K3ListView</class>
      <extends>Q3ListView</extends>
    </customwidget>
  </customwidgets>
  <includes>
    <include location="local" >kurlrequester.h</include>
    <include location="local" >klineedit.h</include>
    <include location="local" >kpushbutton.h</include>
    <include location="local" >kpushbutton.h</include>
    <include location="local" >k3listview.h</include>
    <include location="local" >kpushbutton.h</include>
    <include location="local" >kpushbutton.h</include>
    <include location="local" >kcombobox.h</include>
    <include location="local" >klineedit.h</include>
    <include location="local" >kpushbutton.h</include>
  </includes>
</ui>
